build.gradle

```diff
@@ -75,35 +68,6 @@ subprojects {
 
     apply plugin: 'io.spring.dependency-management'
```

Comment:
We don't need the `io.spring.dependency-management` plugin
We can remove the lines:
```groovy
...
    id 'io.spring.dependency-management' version '1.0.6.RELEASE' apply false
...
    apply plugin: 'io.spring.dependency-management'
...
    apply plugin: 'io.spring.dependency-management'
```